[2023-03-13 22:10:01,564] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.create_table scheduled__2023-03-13T22:05:00+00:00 [queued]>
[2023-03-13 22:10:01,574] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.create_table scheduled__2023-03-13T22:05:00+00:00 [queued]>
[2023-03-13 22:10:01,575] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 22:10:01,575] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-13 22:10:01,576] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 22:10:01,586] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2023-03-13 22:05:00+00:00
[2023-03-13 22:10:01,592] {standard_task_runner.py:52} INFO - Started process 7289 to run task
[2023-03-13 22:10:01,596] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'create_table', 'scheduled__2023-03-13T22:05:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpe7cinstz', '--error-file', '/tmp/tmp2t11sbbm']
[2023-03-13 22:10:01,606] {standard_task_runner.py:80} INFO - Job 138: Subtask create_table
[2023-03-13 22:10:01,690] {task_command.py:369} INFO - Running <TaskInstance: test.create_table scheduled__2023-03-13T22:05:00+00:00 [running]> on host 5282528dd996
[2023-03-13 22:10:01,748] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-13T22:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-13T22:05:00+00:00
[2023-03-13 22:10:01,765] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2023-03-13 22:10:01,771] {dbapi.py:208} INFO - Running statement: 
            create table if not exists users(
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-03-13 22:10:01,773] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2023-03-13 22:10:01,784] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=create_table, execution_date=20230313T220500, start_date=20230313T221001, end_date=20230313T221001
[2023-03-13 22:10:01,809] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-13 22:10:01,843] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
